overlord/ifacestate/helpers.go

```diff
@@ -310,6 +310,26 @@ func (m *InterfaceManager) reloadConnections(snapName string) ([]string, error)
 	return result, nil
 }
 
+func (m *InterfaceManager) setupPhasedSecurity(task *state.Task, snaps []*snap.Info, opts []interfaces.ConfinementOptions) error {
+	st := task.State()
+
+	// Setup all affected snaps, start with the most important security
+	// backend and run it for all snaps. See LP: 1802581
```

Comment:
Might be worth clarifying the comment and say the ordering is established at init time (and maintained in the repo).

Reply:
indeed, this comment probably needs some expanding, important doesn't convey the fact that some backend behavior depends on the work of some other backend